src/components/views/messages/MAudioBody.js

```diff
@@ -96,9 +96,13 @@ export default class MAudioBody extends React.Component {
 
         const contentUrl = this._getContentUrl();
 
+        const shouldAutoPlay = MatrixClientPeg.get().credentials.userId !==
+          this.props.mxEvent.sender.userId &&
+          content.autoplay;
+
         return (
             <span className="mx_MAudioBody">
-                <audio src={contentUrl} controls />
+                <audio src={contentUrl} controls autoPlay={shouldAutoPlay} />
```

Comment:
I don't think we can allow auto-playing audio globally: there's too much scope for abuse. This would also presumably play all files as you scrolled past them in the backlog.